.travis.yml

```
@@ -0,0 +1,14 @@
+sudo: false
+language: go
+go:
+ - "1.13.x"
+script:
+  - go get ./...
+  - go get golang.org/x/lint/golint
+  - go get github.com/axw/gocov/gocov
+  - GO111MODULE=off go get gopkg.in/matm/v1/gocov-html
+  - golint  -set_exit_status ./...
+  - go vet ./...
+  - ./scripts/test.sh -i
```

Do we need the `-i` argument?
```suggestion
  - ./scripts/test.sh
```